packages/next/src/server/node-environment-extensions/utils.tsx

```diff
@@ -44,8 +44,22 @@ export function io(expression: string, type: ApiType) {
                 'Unknown expression type in abortOnSynchronousPlatformIOAccess.'
               )
           }
+
           const errorWithStack = new Error(message)
 
+          if (process.env.NODE_ENV !== 'production') {
+            const ownerStack = workUnitStore.captureOwnerStack!()
```

Comment:
This doesn't exist if you build the app using `NODE_ENV=test`.

Reply:
https://github.com/vercel/next.js/pull/81370

Reply:
Thanks for the swift fix 👍